DuckDuckGo/AppDelegate.swift

```diff
@@ -52,12 +52,6 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         // assign it here, because "did become active" is already too late and "viewWillAppear"
         // has already been called on the HomeViewController so won't show the home row CTA
         AtbAndVariantCleanup.cleanup()
-        DefaultVariantManager().assignVariantIfNeeded { variantManager in
-            // perform first time install logic here:
-            if variantManager.isSupported(feature: .favoritesByDefault) {
-                AppUserDefaults().homePage = .centerSearchAndFavorites
-            }
-        }
```

Comment:
Won't that prevent the app from assigning any variant?

Reply:
Err yeah - was a bit too eager with the delete button there.  Good catch.